web/public/css/header.less

```diff
@@ -133,26 +133,26 @@
 }
 
 .cd-color-1, html .container .cd-color-1 {
-  background-color: #61C93F;
+  background-color: #61C93F !important;
```

Comment:
That's more like "I make it work, the redesign of this process is supposed to be in this sprint and we're 2 days late from the initial deployment" :)